src/components/AppTile.jsx

```diff
@@ -13,7 +13,7 @@ export class AppTile extends Component {
     const displayName = applications.getAppDisplayName(app, lang)
     const appHref = app.links && app.links.related
     return (
-      <AppLinker slug={app.slug} href={appHref}>
+      <AppLinker slug={app.slug} href={appHref} app={app}>
```

Comment:
nit: it could be a great occasion to add one test on this file.

Reply:
Same answer as for https://github.com/cozy/cozy-ui/pull/2015#discussion_r787014463 😛

Reply:
ok !